config/locales/views/groups/en.yml

```
@@ -22,11 +22,14 @@ en:
       edit_name_btn: "Edit Name"
       primary_mentor_heading: "Primary Mentor:"
       mentor_name_heading: "MENTORS:"
-      group_members_heading: "MEMBERS"
+      group_members_heading: "MEMBERS:"
       add_members_btn: "+ Add Members"
       add_mentors_btn: "+ Add Mentors"
+      no_mentors: "There are no mentors in this group, click + Add Member button to add members."
+      no_members: "There are no members in this group, click + Add Member button to add members."
+      no_assignments: "There are no assignments in the group, click + Add Assignments button to start an assignment."
```

```suggestion
      no_mentors_html: "There are no mentors in this group, click <strong>+ Add Mentors</strong> button to add mentors."
      no_members_html: "There are no members in this group, click <strong>+ Add Members</strong> button to add members."
      no_assignments_html: "There are no assignments in the group, click <strong>+ Add Assignments</strong> button to start an assignment."
```

Try this one! also do this for `hi.yml`